modules/apps/data-engine/data-engine-test/src/testFunctional/tests/documentsadministration/fields/DMColorField.testcase

```diff
@@ -37,12 +41,24 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98070. As a Developer, I want to Implement the color field type"
-	@ignore = "true"
 	@priority = "5"
 	test DeleteColorFieldOnBuilder {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98070 DeleteColorFieldOnBuilder pending implementation
+		DMDocumentType.add(
+			dmDocumentTypeDescription = "DM Document Type Description",
+			dmDocumentTypeFieldNames = "Color",
+			dmDocumentTypeName = "DM Document Type Name");
+
+		LexiconEntry.gotoEntryMenuItem(
+			menuItem = "Edit",
+			rowEntry = "DM Document Type Name");
+
+		DataEngine.deleteField(fieldFieldLabel = "Color");
+
+		AssertElementNotPresent(
+			key_fieldFieldLabel = "Color",
+			locator1 = "DDMEditStructure#FORM_FIELD_CONTAINER");
```

Comment:
hi @thaisfrota , could you please validate the action persistence? 

In this case, you need to have two fields added, then you remove one of them, save the document, reopen it and make sure the deleted field is not there.

Thanks!